BitbyBitBlog/components/Blogpostpreview.razor

```diff
@@ -0,0 +1,45 @@
+﻿<div class="previewContianer">
+    
+    <img src="../Images/download.jpg"/>
```

Comment:
this was a great image to use

Reply:
Thanks maybe next time i send something spicy!!!!!